java/client/src/main/java/glide/api/commands/ServerManagementCommands.java

```diff
@@ -180,6 +180,34 @@ public interface ServerManagementCommands {
      */
     CompletableFuture<String> flushall(FlushMode mode);
 
+    /**
+     * Deletes all the keys of the currently selected database. This command never fails.
+     *
+     * @see <a href="https://valkey.io/commands/flushdb/">valkey.io</a> for details.
+     * @return <code>OK</code>.
+     * @example
+     *     <pre>{@code
+     * String response = client.flushdb().get();
+     * assert response.equals("OK");
+     * }</pre>
+     */
+    CompletableFuture<String> flushdb();
```

Comment:
Not sure if it's worth it but we could have the new APIs in this file to be in a different interface `ServerManagementBaseCommands` that would be implemented in `BaseClient` to avoid duplication.